api/src/controllers/inscription-goal.ts

```diff
@@ -104,7 +104,7 @@ router.get(
       if (!canViewInscriptionGoals(req.user)) return res.status(403).send({ ok: false, code: ERRORS.OPERATION_UNAUTHORIZED });
 
       const { department, cohort } = params;
-      const completionObjectif = await getCompletionObjectifs(department, cohort);
+      const completionObjectif = await getCompletionObjectifs(department, cohort, INSCRIPTION_GOAL_LEVELS.DEPARTEMENTAL);
```

Comment:
```suggestion
      const completionObjectif = await getCompletionObjectifs(department, cohort, INSCRIPTION_GOAL_LEVELS.DEPARTEMENTAL);
```
tu es sur que ici on ne veux pas simplement savoir si on a dépassé les objectifs ?
si l'objectif "level" est a région, il doit quand meme retourné qu'on a atteint les objectif pour le département.

Reply:
Vu que la route parle spécifiquement des départements, ça me semblerait bizarre qu'on renvoie des infos de région. Mais peut-être qu'il faut oui

Reply:
Je vais voir aves Thomas. Mais c'est moins complexe de pas faire d'exception et tout baser sur le parametrage dynamique oui. Je vérifie les impacts qd même